beard/clustering/blocking.py

```diff
@@ -99,52 +158,38 @@ def _validate(self, X, blocks):
 
     def _fit(self, X, y, blocks):
         """Fit base clustering estimators on X."""
+        def _grouper(jobs, seq):
```

Comment:
I know you like it, but we couldnt we simply directly import `sklearn.ensemble.base._partition_estimators`? This way we would get rid of some more code :) (Less is better)